okhttp/src/main/java/com/squareup/okhttp/ConnectionSpec.java

```diff
@@ -219,7 +219,7 @@ private ConnectionSpec supportedSpec(SSLSocket sslSocket) {
     private String[] tlsVersions;
     private boolean supportsTlsExtensions;
 
-    Builder(boolean tls) {
+    public Builder(boolean tls) {
```

Comment:
I would wait for @swankjesse to weigh in on the change as whole. This code is his territory!